.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+# To get started with Dependabot version updates, you'll need to specify which
```

Comment:
We're using renovate for these type of updates currently, did we look into https://docs.renovatebot.com/modules/manager/devcontainer/?

Reply:
Good catch, thanks. This was auto-generated and I missed it. Will switch to renovate.

Reply:
1e5f15ff623f9aca9b69771d027038ac45b5d477

Reply:
Have to add the packageManager too, our default has everything off

Reply:
I think you meant enabling here 9913e71ebc67b4e45bef396dab8eab50ad7d1d32